docs/source-pytorch/fabric/advanced/distributed_communication.rst

```diff
@@ -0,0 +1,240 @@
+:orphan:
+
+###########################################
+Communication between distributed processes
+###########################################
+
+With Fabric you can easily access information about a process or send data between processes with a standardized API and agnostic to the distributed strategy.
```

Comment:
```suggestion
With Fabric, you can easily access processes or send data between processes with a standardized API that is agnostic to the selected distributed strategy.
```

Reply:
you can't actually access the process. So I prefer @awaelchli formulation!